AUTHORS

```diff
@@ -26,12 +26,12 @@ ckarlof
 Dan Callahan
 Danny Amey
 Danny Coates
-Dave Justice
```

Comment:
I think this AUTHORS reorder happens because someone has an older version of git on their machine,

@clouserw and @vbudhram could you sync up on Git Versions? :)